botorch/generation/sampling.py

```diff
@@ -254,82 +252,101 @@ def __init__(
         objective: Optional[MCAcquisitionObjective] = None,
         posterior_transform: Optional[PosteriorTransform] = None,
         replacement: bool = True,
-        minimize_constraints_only: bool = False,
```

Comment:
Should we deprecate this arg so this doesn't cause hard failures in other code?

Reply:
I think this should be ok since this functionality probably has no users (that would have likely resulted in Github issues since it wasn't working as intended).